frontend/packages/ceph-storage-plugin/integration-tests-cypress/views/multiple-pool.ts

```diff
@@ -0,0 +1,70 @@
+import { NS } from '../utils/consts';
+import { modal } from '../../../integration-tests-cypress/views/modal';
+
+// Create storage class from
+export const storagePoolDropdown: string = 'pool-dropdown-toggle';
+
+// Create new pool form
+export const poolNameTextBox: string = 'new-pool-name-textbox';
+export const replicaDropdown: string = 'replica-dropdown';
+export const confirmAction: string = 'confirm-action';
+
+// Pool status
+export const emptyStateBody: string = 'empty-state-body';
+
+export const poolMessage = {
+  PROGRESS:
+    'The creation of an OCS storage cluster is still in progress or have failed please try again after the storage cluster is ready to use.',
+  POOL_START: (poolName) => `Pool ${poolName} creation in progress`,
+  POOL_DUPLICATED: (poolName) => `Pool "${poolName}" already exists`,
+  POOL_CREATED: (poolName) => `Pool ${poolName} was successfully created`,
+};
+
+export const storagePool = {
+  prepareStorageClassForm: () => {
+    // Select provisioner
+    cy.byTestID('storage-class-provisioner-dropdown')
+      .click()
+      .get('#openshift-storage\\.rbd\\.csi\\.ceph\\.com-link')
```

Comment:
```<a href="#" id="openshift-storage.rbd.csi.ceph.com-link" data-test="dropdown-menu-item-link" class="pf-c-dropdown__menu-item">openshift-storage.rbd.csi.ceph.com</a>```
The element as rendered in the DOM
Perhaps something like
`cy.byTestID('dropdown-menu-item-link').contains('openshift-storage.rbd.csi.ceph.com')`